resources/idam_stub_get_details_custom.json

```diff
@@ -1,7 +1,7 @@
 {
   "request": {
     "method": "GET",
-    "urlPath": "/details"
+    "urlPath": "/o/userinfo"
```

Comment:
can you edit body as well. Property names are not same.

Reply:
Fixed.. Please review

Reply:
You might need to add `depends_on` idam-api for the backend docker config. With openId backend apps talk to idam-api  (/o/.well-known/openid-configuration) startup

Reply:
I don't think that is needed.... this is only for stub service...

Reply:
nope..NimbusJWTDecoder in the definition-store /data-store makes a call to that api. You can test that in the ccd-docker with idam dockers.